### PR TITLE
Sync branches when mirroring

### PR DESCRIPTION
**Pull request by lunny**
Caused by #22743

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

File: modules/git/git.go [hunk 1]
```diff
@@ -359,7 +358,7 @@ func configSetNonExist(key, value string) error {
 func configAddNonExist(key, value string) error {
 	_, _, err := NewCommand(DefaultContext, "config", "--global", "--get").AddDynamicArguments(key, regexp.QuoteMeta(value)).RunStdString(nil)
 	if err == nil {
-		// already exist
+		// already exist                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   ,
```
Comment (wxiaoguang) [inline modules/git/git.go]: unrelated change?

Comment (lunny) [reply modules/git/git.go]: I think it's editors behaviour.

**Review by delvh — APPROVED** [review]

File: modules/git/git.go (hunk 1 above)
Comment (KN4CK3R) [reply modules/git/git.go]: But there are lots of spaces and a `,`

Comment (KN4CK3R) [inline modules/git/git.go]:
```suggestion
		// already exist
```

**Review by wolfogre — APPROVED** [review]
